docs/performance/database-indexes.rst

```diff
@@ -0,0 +1,20 @@
+Database indexes
+================
+
+Indexing tables allows you to decrease fetching time from database.
+
+As an example lets take a look on customers list
+
+Default index page is sorted by registration date, to create table index all you need to do is modify `Customer` Entity and add index using annotations.
+`ORM annotations <https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/annotations-reference.html#annref_haslifecyclecallbacks>`_
+
+.. code-block:: php
+
+    /**
+     * @Entity
+     * @Table(name="customer",indexes={@Index(name="created_at_index", fields={"created_at"})})
+     */
```

Comment:
```suggestion
     <?php

    // src/Entity/Customer/Customer.php
    
    declare(strict_types=1);
    
    namespace App\Entity\Customer;
    
    use Doctrine\ORM\Mapping as ORM;
    use Sylius\Component\Core\Model\Customer as BaseCustomer;
    
    /**
     * @ORM\Entity
     * @ORM\Table(name="customer",indexes={@Index(name="created_at_index", fields={"created_at"})})
     */
    class Customer extends BaseCustomer
    {
    }

```

Reply:
`// src/Entity/Customer/Customer.php` is not even needed as we have namespace.